Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-Article [XX] (Amendment 20 - Terms of President, Vice President, Members of Congress: Presidential Vacancy)
-1: The terms of the President and Vice President shall end at noon on the 20th day of January, and the terms of Senators and Representatives at noon on the 3d day of January, of the years in which such terms would have ended if this article had not been ratified; and the terms of their successors shall then begin.   affects 5
+2: It is the Right of all People to associate or decline to associate with other People; and to form contracts; and to assemble in public places, and in private places with the assent of the owner of the place; and to engage in commerce, or refuse to engage in commerce, as fits their individual needs and desires; and the Responsibility of all People to ensure that their associations work toward the betterment of all associated, without causing harm to those not associated.
 
-2: The Congress shall assemble at least once in every year, and such meeting shall begin at noon on the 3d day of January, unless they shall by law appoint a different day.   affects 5
+3: It is the Right of all People to speak as they see fit, and to distribute their thoughts and speech through whatever means they are able to peaceably attain, and to hear or otherwise receive the speech and thoughts of others, as they are so made available; and the Responsibility of all People to speak Truth, as best as can be perceived.
 
-3: If, at the time fixed for the beginning of the term of the President, the President elect shall have died, the Vice President elect shall become President. If a President shall not have been chosen before the time fixed for the beginning of his term, or if the President elect shall have failed to qualify, then the Vice President elect shall act as President until a President shall have qualified; and the Congress may by law provide for the case wherein neither a President elect nor a Vice President elect shall have qualified, declaring who shall then act as President, or the manner in which one who is to act shall be selected, and such person shall act accordingly until a President or Vice President shall have qualified.   affects 9    affects 14
+4: It is the Right of all People to petition their Government for redress of grievances; and the Responsibility of all People to watch their Government and ensure its good behavior, and to attempt to peaceably resolve intrusions by Government on the Rights of the People, before rising in arms against such an intrusion; and to rise in arms against the Government when peaceful resolutions become impossible.
```

Comment:
Again I make reference in my commentary to the Four Boxes in Defense of Liberty. Only when the first three boxes (Soap, Ballot, and Jury) are exhausted should the fourth be opened. Nonetheless, the people have a responsibility to make full use of all four, when necessary.